Simple solvers/RM.py

```diff
@@ -19,12 +19,12 @@
 root1NewMethod=(averageOfRoots)-(distanceFromRoot) # This is the first answer given by the new method 
 root2NewMethod=(averageOfRoots)-(distanceFromRoot2) # This is the second answer given by the new method (If there is a second root)
 if (discriminant==0)or(discriminant>0):
-    if (root1QuadraticFormula==root1NewMethod): #To make sure that the quadratic formula and new method give the same result for the first root
-        answer1=round(root1QuadraticFormula, 2) #root1QuadraticFormula rounded to two decimal places
-        print(answer1) 
+    if (root1QuadraticFormula==root1NewMethod): # To make sure that the quadratic formula and new method give the same result for the first root

```

Comment:
Really long lines are hard to read because side-scrolling sucks.  Consider moving comments to the line above the code, with a blank line before the comment line.
```suggestion

    # To make sure that the quadratic formula and new method give the same result for the first root
    if (root1QuadraticFormula==root1NewMethod): 
```

Reply:
This comment applies to ALL your inline comments in this file.